src/PrestaShopBundle/Form/Admin/Improve/International/Currencies/CurrencyType.php

```diff
@@ -69,36 +78,100 @@ public function __construct(array $allCurrencies, $isShopFeatureEnabled)
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('iso_code', ChoiceType::class, [
+            ->add('selected_iso_code', ChoiceType::class, [
                 'choices' => $this->allCurrencies,
                 'choice_translation_domain' => false,
+                'required' => false,
+                'placeholder' => '--',
+            ])
+            ->add('is_custom', CheckboxType::class, [
+                'required' => false,
+                'label' => $this->trans('Create new', 'Admin.International.Feature'),
```

Comment:
Create new what? currency?

Reply:
Yes, I replace with `Create custom currency`

Reply:
Sorry to insist but I don't get the difference between the words 'new' and 'custom', can you explain why are we choosing 'custom' over 'new' here? We rarely saw this word employed in the back office so far, that's why I'm asking. ;-)

Reply:
@MatShir, perhaps you will be able to help me understand it!

Reply:
A custom currency is a currency that doesn't really exist, not registered as a real ISO currency from a country like Euro, Dollar, ...
This feature can be used to create a virtual currency, or a locale currency Which is why I considered using "custom" was more appropriate than "new" in this context

Reply:
I got it, thanks. I would find it more user-friendly to add a tooltip to explain what we mean when we talk about custom currencies.
+ @colinegin

Reply:
Not bad, I prefer "custom" to "new" but I am afraid that some user will understand it as editing the currency. Also, it seems hard to translate it even in french for us. 
And does custom currency feel like a custom currency for the user or an official one his point of view?

Wdyt @jolelievre @TristanLDD ? 

Nevertheless, we will start with "custom" and try to contact interested user on cryptocurrencies or local currencies to do some user test.